public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -312,9 +317,9 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
   }
 
   render() {
-    const { dashboard, initError, queryParams, theme } = this.props;
+    const { dashboard, initError, queryParams, theme, isPublic } = this.props;
     const { editPanel, viewPanel, updateScrollTop } = this.state;
-    const kioskMode = getKioskMode();
+    const kioskMode = !isPublic ? getKioskMode() : KioskMode.Full;
```

Comment:
Why is public changing kiosk mode?

Reply:
Kiosk mode hides the dashboard header, which we wanted to exclude for our first version of public dashboards

Reply:
There are a lot of actions in the header that aren't relevant to public dashboards for our first version, like saving the dashboard, adding panels, and changing the dashboard settings. We want to keep public dashboards a simple as possible for our first iteration.

Reply:
ah, but then you will loose time range picker. but I understand. You using full kiosk mode when viewing a public dashboard. That means the user will not even see the name of the dashboard. 

> like saving the dashboard, adding panels, and changing the dashboard settings.

You can control what actions are available by setting the dashboard.meta flag.

Reply:
For the next iteration of this we'll be adding the dashboard title